ts/features/fci/screens/valid/FciDocumentsScreen.tsx

```diff
@@ -136,29 +135,31 @@ const FciDocumentsScreen = () => {
 
   const onCancelPress = () => present();
 
-  const cancelButtonProps: ButtonSolidProps = {
+  const cancelButtonProps: ComponentProps<
+    typeof FooterActionsInline
+  >["startAction"] = {
     onPress: onCancelPress,
-    label: I18n.t("features.fci.documents.footer.cancel"),
-    accessibilityLabel: I18n.t("features.fci.documents.footer.cancel")
+    label: I18n.t("features.fci.documents.footer.cancel")
   };
 
-  const continueButtonProps: ButtonSolidProps = {
+  const continueButtonProps: ComponentProps<
+    typeof FooterActionsInline
+  >["endAction"] = {
     onPress: onContinuePress,
-    label: I18n.t("features.fci.documents.footer.continue"),
-    accessibilityLabel: I18n.t("features.fci.documents.footer.continue")
+    label: I18n.t("features.fci.documents.footer.continue")
   };
 
-  const keepReadingButtonProps: ButtonSolidProps = {
+  const keepReadingButtonProps: ComponentProps<
+    typeof FooterActionsInline
+  >["endAction"] = {
     onPress: () => pointToPage(totalPages),
-    label: I18n.t("global.buttons.continue"),
-    accessibilityLabel: I18n.t("global.buttons.continue")
+    label: I18n.t("global.buttons.continue")
   };
 
-  const secondaryButtonProps: BlockButtonProps = {
-    type: currentPage < totalPages ? "Outline" : "Solid",
-    buttonProps:
-      currentPage < totalPages ? keepReadingButtonProps : continueButtonProps
-  };
+  const secondaryButtonProps: ComponentProps<
```

Comment:
What do you think about renaming it to `endActionButtonProps`?

Reply:
Addressed here → [`435f6e6` (#6477)](https://github.com/pagopa/io-app/pull/6477/commits/435f6e6bfe7c8f0024986b8693c21fd5c69bc28d)